purchases/src/main/kotlin/com/revenuecat/purchases/FeatureLockUtils.kt

```diff
@@ -0,0 +1,11 @@
+package com.revenuecat.purchases
+
+@Suppress("MaxLineLength")
+internal enum class LockedFeature(val isLocked: Boolean, val lockedMessage: String) {
+    ObserverMode(true, "Observer mode is not supported in this version. Please disable it or use the latest stable version to use this."),
+    AmazonStore(true, "Amazon store is not supported in this version. Please use the Google store or use the latest stable version to use this."),
+    SyncPurchases(true, "Syncing purchases is not supported in this version. Please use the latest stable version to use this."),
+    InAppPurchasing(true, "Purchasing INAPP products is not supported in this version. Please use the latest stable version to use this.")
```

Comment:
I believe these are all the features disabled for the first beta. Do lmk if you think I'm missing any!

Reply:
cool 😻

Reply:
Hmm well, other than removing these methods entirely, I'm not sure of other ways to fail during compile time. And with that option, I'm concerned of us getting existing customers into the beta with their implementation done and having us remove some of these methods causing a bigger pain to migrate it... Which might be acceptable to avoid issues early... Another issue is that we can't really detect trying to purchase OTP at compile time, so that would still have to be a runtime check, and it might be better to keep it consistent? 

I think reporting the issue at runtime would be enough if we document this prominently in the beta docs but happy to chat if you think removing part of the API would be better

Reply:
ah that's a good point about OTP. no, i'm okay with this, just thought it was worth thinking about!